packages/react-native-bridge/android/build.gradle

```diff
@@ -137,7 +158,9 @@ repositories {
 }
 
 dependencies {
-    implementation project(':@wordpress_react-native-aztec')
+    // embed is a method from the fat-aar plugin that will the Aztec project
+    // dependency into the generated AAR
+    embed project(':@wordpress_react-native-aztec')
```

Comment:
The change here from `implementation` to `embed` seems to break the build-from-source build. I'm not sure why that is, but it's simple enough to workaround since we could just use an if statement to use `embed` or `implementation` depending on the build type (we already do that for a lot of other dependencies just a few lines below here).